libs/application/templates/parental-leave/README.md

```diff
@@ -2,195 +2,108 @@
 - `api`
-  - Fetch secrets using: `yarn get-secrets api`
+  - Use: `yarn get-secrets api`
 - `application-system-form`
 
-Once you have everything running you can navigate to [http://localhost:4200/umsoknir/faedingarorlof](http://localhost:4200/umsoknir/faedingarorlof) and start developing.
-
-### Local database
-
-By setting up the application-system you'll have created a local postgres database on a docker image, if you haven't already you should setup a tool to interact with your database. For example [pgAdmin](https://www.pgadmin.org/download/).
-
-You’ll find the relevant connection information in [the docker compose file](../../../../apps/application-system/api/docker-compose.base.yml).
-
-## Investigating errors
-
-When investigating errors on our enviornments we have two logging services. `Datadog` for backend services and `Sentry` for the client.
-
-### Datadog
-
-- env: `dev` | `staging` | `prod`
-- service: `api` | `application-system-api`
-
-### Sentry
-
-- Can select `dev` / `staging` / `prod` from the environment dropdown
-- Can filter by url with id to find a single application experiencing errors
-  - `url:https://island.is/umsoknir/:id`
-
-## Future work
-
-### Screens for users that are no longer assignees
-
-When the application is opened/refreshed by a user that was an assignee but no longer is, we might want to show them something - we get their national registry id so we could see if it is the other parent visting or the employer, even though they are not an assignee anymore and show them something relevant like "The application is no longer in a state you can interact with". This is too specific for the application system to know and handle, although the application system should show some message to a user visiting an application that is not an assignee.
+Access at [http://localhost:4200/umsoknir/faedingarorlof](http://localhost:4200/umsoknir/faedingarorlof).
 
-### Localisation in emails
+### Local Database
 
-Emails are currently hardcoded, need to pass in localisation functions to be able to use translation strings. Also need to sync with service-portal to know what locale the user prefers.
+A local PostgreSQL database is created via Docker. Tools like pgAdmin can be used for database interaction.
 
-### Other parent asks for personal allowance
+## Error Investigation
 
-Other parent should be able to ask the primary parent for personal allowance. Now only primary parent can ask the other parent for personal allowance.
+Use `Datadog` for backend and `Sentry` for client error logging.
 
-### Non-custodial parent
+## Future Work
 
-If primary parent doesn't give non-custodial parent consent that they is allowed right of access during the parental leave (grant) they should not be able to apply for parental leave (grant).
+- Screens for users who are no longer assignees.
+- Localisation in emails.
+- Allow secondary parent to request personal allowance.
+- Restrict non-custodial parents lacking consent from applying.
```

Comment:
This last point has already been implemented.  Can be removed. Should we do that?

Reply:
Up to you, this is your service. Just push to this branch :heart: